biomage/rds/start_port_forwarding.py

```diff
@@ -0,0 +1,184 @@
+import click
+
+from ..utils.constants import STAGING
+
+from subprocess import PIPE, run
+
+@click.command()
+
+@click.option(
+    "-i",
+    "--input_env",
+    required=False,
+    default=STAGING,
+    show_default=True,
+    help="Input environment of the RDS server.",
+)
+
+# Disabled, only 5432 works for now
+# @click.option(
+#     "-p",
+#     "--local_port",
+#     required=False,
+#     default=5432,
+#     show_default=True,
+#     help="Local port from which to connect.",
+# )
+
+def start_port_forwarding(input_env, local_port = 5432):
+    """
+    Sets up a port forwarding session for the rds server in a given environment.\n
+
+    E.g.:
+    biomage rds start-port-forwarding -i staging
+    """
+
+    run(f"./biomage/rds/start_port_forwarding.sh {input_env} {local_port}", shell=True)
+
+# Everything that is commented out here is a python version in case it is useful if we try to reimplement it in python instead
+# def start_port_forwarding(input_env, local_port):
+#     """
+#     Sets up a port forwarding session for the rds server in a given environment.\n
+
+#     E.g.:
+#     biomage rds start_port_forwarding -i staging -p 5432
+#     """
+#     
+#     rds_client = boto3.client("rds")
+#     ec2_client = boto3.client("ec2")
```

Comment:
so is this your attempt to implement it in python and then leaving it for now?

Reply:
If we decide to implement it in Python, will this commented out code be useful to someone who didn't write it?

Reply:
I think so, part of this actually seemed to work in python, and most of the clients I am importing are probably going to be used in a very similar way to how I was trying to use them.

Reply:
maybe I should move all the commented out stuff to a different file? That way it is still around but doesn't add noise the current implementation? Or just move it away from the branch and into the ticket or something?

Reply:
we don't squash merge so you can just remove it, and if you need it you can check the commit history to get it back